tinycrypt/ecc.c

```diff
@@ -427,7 +428,7 @@ void ecc_wait_state_reset(ecc_wait_state_t *ws)
 		return;
 
 	ws->i = 0;
-	g_rng_function(ws->delays, sizeof(ws->delays));
```

Comment:
What rng was used here before?

Reply:
@AndrzejKurek , g_rng_function is a pointer to function set by user for entire ECC module usage. It should be genrator that implemented according to DRBG standart. In our project HMAC DRBG is used and HMAC DRBG was called to generate delays

Reply:
OK, so someone configured HMAC_DRBG to be the g_rng_function, right? Why can't we change it there, to be the `platform_random_buf`, but implemented on their side using `mbedtls_platform_random_uint16`?

Reply:
@AndrzejKurek , because this g_rng_function is used for important crypto operations in ECC and have to be standard DRBG in all other places besides delay generations

Reply:
oh, ok, now I get it! thanks.